network_diffusion/tests/mln/test_mlnetwork.py

```diff
@@ -18,25 +18,60 @@ def copy_helper(original_network, copied_network):
         assert nx.utils.graphs_equal(copied_layer, orig_layer) is True
 
 
+exp_actors = {
+    "Pazzi",
+    "Tornabuoni",
+    "Peruzzi",
+    "Salviati",
+    "Medici",
+    "Lamberteschi",
+    "Guadagni",
+    "Ginori",
+    "Castellani",
+    "Bischeri",
+    "Barbadori",
+    "Ridolfi",
+    "Strozzi",
+    "Albizzi",
+    "Acciaiuoli",
+}
+exp_layers = {"business", "marriage"}
+
+
 class TestMultilayerNetwork(unittest.TestCase):
     """Test MultilayerNetwork class."""
 
     def setUp(self):
         """Set up most common testing parameters."""
-        self.network = MultilayerNetwork.from_mpx(
+        self.network_f = MultilayerNetwork.from_mpx(
```

Comment:
(2) In my opinion the florentine/florentine_network would be clearer

Reply:
applied